internal/cmd/docker/push.go

```diff
@@ -0,0 +1,129 @@
+package docker
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/registry"
+	"github.com/docker/docker/client"
+	cmds "github.com/saucelabs/saucectl/internal/cmd"
+	"github.com/saucelabs/saucectl/internal/segment"
+	"github.com/saucelabs/saucectl/internal/usage"
+	"github.com/spf13/cobra"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func PushCommand() *cobra.Command {
+	var registryPushTimeout time.Duration
+	var quiet bool
+
+	cmd := &cobra.Command{
+		Use:          "push <IMAGE_NAME>",
```

Comment:
Yet a new casing standard? Can we please use something that's already wide spread in the code base, or standardize one across our CLI?

Just pulling some examples from _our existing code base_:
- `upload <jobID> <filename>`
- `download <jobID> <file-pattern>`
- `delete <fileID>`
- `download fileID`
- `upload filename`

^ As you can see, it's _already_ not great. And this PR is going to add to the list.

Reply:
Updating ✍️